flake.lock

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
The flake.lock should not be commited in this repo, because it is a library (see home-manager, nixpkgs).